### ISSUE TITLE
Volume on vega.xyz does not specify within what timeframe

### ISSUE THREAD

**Issue by daunatv** [vegaprotocol/vega.xyz#835]
## Description

On https://vega.xyz there is a section that shows Volume, Total markets, TVL, AVG Blocktime.
The Volume does not specify what timeframe, it could be assumed total volume since the start but I would assume it is 24 hrs or epoch? 

## Screenshot
<img width="1317" alt="Screenshot 2023-08-02 at 09 05 28" src="https://github.com/vegaprotocol/vega.xyz/assets/100769129/c76b027f-151c-4d19-8ee9-928b19b1df62">

## Tasks

1. Hide the currently derived daily volume figure.  We will bring this back later when have more daily volume, so dont lose the code.
2. Add a new value instead for Total Volume.  I think we will need to derive this basically using the same method as before but doing it across all epochs. 

1. Query /api/v2/rewards/epoch/summaries
2. ~Filter for `epoch` = current epoch~ do not filter for epoch
3. Filter for `ACCOUNT_TYPE_FEES_INFRASTRUCTURE`
4. For the `amount` given for each `assetId` retrieve the value of `decimals` from the [assets API](https://api.vega.community/api/v2/assets)
5. Derive the "human readable amount" as `amount` / 10^ `decimals`
6. Divide each human readable amount by network parameter `market.fee.factors.infrastructureFee`
7. Use the Coingecko API at https://api.coingecko.com/api/v3/simple/price?ids=vega-protocol,tether,usd-coin,weth&vs_currencies=usd&include_last_updated_at=true
8. Multiply each value by the corresponding `usd` value in the API above (Use coingecko API key mapping table below to identify the relevant CCY pair) 
9. Add these up to get the Total Volume in USD

[Coingecko API mapping file](https://docs.google.com/spreadsheets/d/1PrVEnUAqRSof2BhWKZSFo5JaBIvVjDrZmptaMvKJ3xc/edit#gid=0)

In other words, use all the same code but dont filter for epoch, and cross your fingers the query still runs reasonably quickly ...

**Comment by JonRay15** [comment]
GOOD POINT 

Should change to 24h volume

**Comment by JonRay15** [comment]
Change of plan ... they actually want to show the total volume to date!

Have updated the description on the ticket

**Comment by goslingo** [comment]
The deploy preview shows testnet data

https://vega-id127tj8z-vegaprotocol.vercel.app/
